tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+#
+#
+# Summary: Test if the Screen Sharing GNOME Desktop functionality works
+#   We currently need to install vino package for that
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    record_info 'vino missing',     'After the installation the screen sharing is not available - vino is missing.';
+    assert_and_click 'upper_right', 'left', 90;
+    assert_and_click 'settings',    'left';
+    assert_and_click 'sharing',     'left';
+    assert_screen 'without_screensharing';
+    send_key 'ctrl-q';
+
+    record_info 'Install vino', 'We are going to install vino.';
+    mouse_hide;
+    ensure_installed 'vino';
+
+    record_info 'Relogin',          'We must log off and back in to see the change.';
+    assert_and_click 'upper_right', 'left';
```

Comment:
you can replace whole block with two lines . take a look into https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/x11/user_gui_login.pm. Or at least see how they implemented and reuse same way.